autoload.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * @author : Gaellan
+ * @author : Marine Sanson
+ */
+
+require "./controllers/AbstractController.php";
+require "./services/Router.php";
+
+require "./models/Admin.php";
+require "./models/Category.php";
+require "./models/Contact.php";
+require "./models/Media.php";
+require "./models/News.php";
+require "./models/Recipe.php";
+require "./models/Order.php";
+require "./models/Product.php";
+require "./models/Variety.php";
+
+require "./managers/AdminManager.php";
+require "./managers/ProductManager.php";
+require "./managers/VarietyManager.php";
+require "./managers/OrdersManager.php";
+require "./managers/CategoryManager.php";
+require "./managers/NewsManager.php";
+require "./managers/RecipeManager.php";
+require "./managers/MediaManager.php";
+require "./managers/ContactManager.php";
+
+require "./controllers/FileUploader.php";
+
+require "./controllers/HomeController.php";
+require "./controllers/AboutController.php";
+require "./controllers/GalleryController.php";
+require "./controllers/ProductController.php";
+require "./controllers/VarietyController.php";
+require "./controllers/ContactController.php";
+require "./controllers/BasketController.php";
+require "./controllers/OrdersController.php";
+require "./controllers/NewsController.php";
+require "./controllers/AdminController.php";
+require "./controllers/AdminMediasController.php";
+require "./controllers/AdminProductsController.php";
+require "./controllers/AdminOrdersController.php";
+require "./controllers/AdminDisplayOrdersController.php";
+require "./controllers/AdminNewsController.php";
+require "./controllers/AdminContactController.php";
+
+$routes = [];
+
+// Read the routes config file
+$handle = fopen("config/routes.txt", "r");
+
+if ($handle) { // if the file exists
+
+    while (($line = fgets($handle)) !== false) { // read it line by line
+
+        $route = []; // each route is an array
+
+        $routeData = explode(" ", str_replace(PHP_EOL, '', $line)); // divide the line in two strings (cut at the " ")
```

Comment:
Si jamais on te pose la question : EOL ici veut dire End Of Line

Reply:
Ha yes merci!!!